src/test/crimson/seastore/test_randomblock_manager.cc

```diff
@@ -61,13 +61,13 @@ struct rbm_test_t :
     config.block_size = DEFAULT_BLOCK_SIZE;
     config.total_size = DEFAULT_TEST_SIZE;
     config.device_id = d_id;
-    return tm_setup();
+    return seastar::now();
   }
 
   seastar::future<> tear_down_fut() final {
     rbm_manager.reset();
     device.reset();
-    return tm_teardown();
```

Comment:
Is there anything depends on `TMTestState` after dropping these?

Reply:
Removed TMTestState.

Reply:
The according #include is also unnecessary.

Reply:
Done.